solutions/pytorch_tests/Makefile

```diff
@@ -0,0 +1,135 @@
+	grep -q -E '1331 passed.*1435 skipped.*36 deselected.*3 xfailed' $@.out
+
+.PHONY: run-test-ops
+run-test-ops: rootfs
+	$(RUN_PYTEST) /workspace/pytorch/test/test_ops.py 2>&1 | tee $@.out
+	grep -q -E '4200 passed.*10069 skipped' $@.out
+
+.PHONY: run-test-torch
+run-test-torch: rootfs
+	$(RUN_PYTEST) /workspace/pytorch/test/test_torch.py -k "not test_RNG_after_pickle and not test_from_file and not test_torch_from_file" 2>&1 | tee $@.out
+	grep -q -E '693 passed.*527 skipped.*3 deselected' $@.out
+
+.PHONY: run-core-tests
+run-core-tests: run-test-autograd run-test-autograd-thread-shutdown run-test-modules run-test-nn run-test-ops run-test-torch
+
+.PHONY: run
+run: run-core-tests
+
+.PHONY: one
+one: rootfs
+	$(RUN_PYTEST) /workspace/pytorch/test/test_nn.py -s -k "test_EmbeddingBag_per_sample_weights_and_no_offsets_cpu_int64_float32"
+
+.PHONY: gdb
+gdb: rootfs
+	$(MYST_GDB) --args $(MYST_EXEC) --max-affinity-cpus=1 rootfs --app-config-path config.json /usr/local/bin/python -m pytest /workspace/pytorch/test/test_autograd.py -v -s -k "test_too_many_grads"
+
+pdb-rootfs: appdir
+	wget https://raw.githubusercontent.com/deislabs/mystikos/main/scripts/mpdb.py
+	cp mpdb.py appdir
+	$(MYST) mkext2 appdir pdb-rootfs
+
+.PHONY: pdb
+pdb: pdb-rootfs
+	$(MYST_EXEC) --max-affinity-cpus=1 rootfs --app-config-path config.json /usr/local/bin/python -m mpdb /usr/local/bin/pytest /workspace/pytorch/test/test_autograd.py -v -s -k "test_too_many_grads"
+
+#################################################
+# Build pytorch C++ lib from github source.
+# Used for local debug.
```

Comment:
We should explain what the Dockerfiles are for in `Dockerfile` and `Dockerfile-build`.

Reply:
Fixed.